packages/server/lib/modes/record.js

```diff
@@ -662,6 +672,8 @@ const createRunAndRecordSpecs = (options = {}) => {
           parallel,
           ciBuildId,
           machineId,
+          framework: testingType === 'component' ? getDevServerAttribute(config?.devServer, 'framework') : undefined,
+          bundler: testingType === 'component' ? getDevServerAttribute(config?.devServer, 'bundler') : undefined,
```

Comment:
I would also expect this logic to get applied when providing the resolved runtime configuration to the run instance results: https://github.com/cypress-io/cypress/blob/framework-cloud-telemetry/packages/server/lib/modes/record.js#L807

Reply:
Yeah I'm a little confused about what this part is used for - we just want the Cloud to be able to store the framework and bundler configured for each instance so that we can query it with Metabase. 

The reason that we differentiate between "custom" and other types of bundlers is so that this is a string value that can be stored in the Cloud db.